cmake/kaldi-native-fbank.cmake

```diff
@@ -0,0 +1,41 @@
+function(download_kaldi_native_fbank)
```

Comment:
Could you replace it with
https://github.com/k2-fsa/sherpa-ncnn/blob/master/cmake/kaldi-native-fbank.cmake

Compiling kaldi-native-fbank from source makes the user's life easier.

---

You need to add something like the following line:
https://github.com/k2-fsa/sherpa-ncnn/blob/0a65dbe30a57e3da70aa8a28d756b51f2b680929/sherpa-ncnn/csrc/CMakeLists.txt#L11
```
target_link_libraries(sherpa-ncnn-core kaldi-native-fbank-core ncnn)
```

That is, you need to link with `kaldi-native-fbank-core `.